exec/jdbc/src/test/java/org/apache/drill/jdbc/PreparedStatementTest.java

```diff
@@ -410,6 +417,155 @@ public void testNonTriggeredQueryTimeout() throws SQLException {
     }
   }
 
+  ////////////////////////////////////////
+  // Query maxRows methods:
+
+  /**
+   * Test for reading of default max rows
+   */
+  @Test
+  public void   testDefaultGetMaxRows() throws SQLException {
+    try(PreparedStatement pStmt = connection.prepareStatement(SYS_OPTIONS_SQL)) {
```

Comment:
Please add spaces here and in other places where needed (and remove where they are excessive, like in the signature of this test).
```suggestion
    try (PreparedStatement pStmt = connection.prepareStatement(SYS_OPTIONS_SQL)) {
```